apps/speed.c

```diff
@@ -1033,11 +1033,12 @@ static int ECDSA_verify_loop(void *args)
     }
     return count;
 }
-
+/* TBC: outlen is globally shared, in a strange way*/
```

Comment:
Ping @agrandi, I think this is to do with the your async changes. Should this be a member of `loopargs_t`?